docs/source/model_design/vivarium_model_components/risk_factors/risk_mediation/index.rst

```diff
@@ -30,4 +30,384 @@ Risk factor mediation
+high systolic blood pressure.
+Draws of the mediation factor are allowed to be negative, which represents the direct pathway
+and the mediated pathway having different effect directions.
+However, if more than 50 of the 1,000 draws were negative, that mediation
+pathway was excluded from GBD.
+
+.. todo::
+  Why does GBD exclude due to negative draws?
+
+This "mediation factor" can be used to calculate values for each of the causal effects represented by arrows
+in the diagram in one of two ways; the following sections describe how to do this.
+
+Unfortunately, GBD mediation factors are inconsistent with the multiplicative application of RRs.
+This is because mediation factors are defined as proportions of the *excess* risk -- the amount the RR is
+greater than 1.
+In effect, GBD's mediation methods can split e.g. an RR of 2.2 into an RR of 1.7 and an RR of 1.6 (since 0.7 + 0.6 = 1.2, which
+is the overall RR's excess risk).
+However, if we apply these two RRs, we end up with an RR of of 1.7 * 1.6 = 2.72, which is higher than
+we started with.
+For this reason, we have to decide whether to be consistent with how we apply RRs
+(and therefore stay calibrated to overall prevalence),
+or to be consistent with GBD's mediation methodology.
+In past models, we have mostly done the latter; we recommend doing the former in future models.
+Both approaches are described below.
+
+.. todo::
+
+  We don't totally understand why GBD models *excess* risk -- while there is a possibility that
+  there is a good reason behind it, we haven't been able to find that reason, so we prefer
+  Approach 1 for now.
+
+If GBD does not report a mediation factor, it may still report one or more of the effects in the diagram.
+The rest can be filled in with external sources.
+
+.. note::
+  Be careful not to interpret GBD RRs as *direct* RRs -- unless there is a mediation factor,
+  the GBD RR will not be adjusted for any mediators.
+
+All our multiplicative math is based on this equation describing how RRs are applied:
+
+.. math::
+
+  RR_\text{r→o} = RR_\text{r→o,direct} * \prod_{m \in M} (RR_\text{m→o})^{\Delta_\text{r→m}}
+
+where :math:`M` is the set of mediators that mediate the relationship between risk :math:`r`
+and outcome :math:`o` and :math:`\Delta_\text{r→m}` is the (assumed linear) increase in :math:`m`
+caused by a one-unit increase in :math:`r`.
+
+.. note::
+  This ignores the concept of a TMREL below which the RR does not apply and risk
+  is flat.
+  This seems to be ignored throughout the GBD's descriptions of its mediation methods, although
+  some risks that are mediators in GBD, such as BMI, clearly have non-zero TMRELs.
+
+.. note::
+  :math:`\Delta_\text{r→m}` only makes sense if both the risk and the mediator are continuous values.
+  Since all of the mediation we have modeled so far has been between continuous
+  risk factors, we haven't yet explored how to make different assumptions (neither has GBD).
```

Comment:
I believe that this is the setting that Pearl calls "Linear Wonderland" in _The Book of Why_

Reply:
On my to-read list! But, I'm guessing we frequently leave "Linear Wonderland" due to RRs, which make many associations in our models log-linear 😞 It's actually only risk-risk effects that are usually linear!

Reply:
In my new PR this is much more fleshed out, though I don't reference Pearl (yet!).